src/api/app/views/webui/request/beta_show_tabs/_build_results.html.haml

```diff
@@ -1 +1,7 @@
-Build results
+%p
+  The build results aren't here yet. For now, you can see them on the request's
+  = link_to('source package', package_show_path(project, package))
+  page. We're working on having them here as soon as possible.
```

Comment:
This is not the request's page. It's the source package.

I would also not make promises like `as soon as possible` :)

Reply:
> This is not the request's page. It's the source package.

The link is on `source package`, so this is already the case.

> I would also not make promises like as soon as possible :)

Good point, I pushed some changes.